examples/asr/librispeech_emformer_rnnt/global_stats.py

```diff
@@ -0,0 +1,78 @@
+"""Generate feature statistics for LibriSpeech training set.
+
+Example:
+python global_stats.py --librispeech_path /home/librispeech
+"""
+
+import json
+import logging
+import pathlib
+from argparse import ArgumentParser
+
+import torch
+import torchaudio
+from utils import GAIN, piecewise_linear_log, spectrogram_transform
+
+logger = logging.getLogger()
+
+
+def parse_args():
+    parser = ArgumentParser(description=__doc__)
```

Comment:
nit: You might need `formatter_class=RawTextHelpFormatter`. Try running `--help` and see if the docstring is printed as you intended.